pkg/workflow/asset/data/schemas/v1/Workflow.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Workflow",
+  "description": "A project nebula workflow definition",
+  "type": "object",
+  "properties": {
+    "apiVersion": {
```

Comment:
I think we may as well add a `kind: Workflow` for consistency, but see below for more...